src/main/python/e2e_experimental_settings.py

```diff
@@ -5,23 +5,23 @@
 import shlex
 import subprocess
 
+
 class Setting(object):
     def __init__(self, label, value_flag_map):
         self.label = label
         self.choice_flags = value_flag_map
-    
+
     def get_settings(self):
         return self.choice_flags.keys()
 
     def get_choice(self, setting):
         return self.choice_flags[setting]
 
     def get_options(self):
-        options = []
-        for key in self.choice_flags.keys():
-            options.append("{}:{}".format(self.label, key))
+        options = [options.append("{}:{}".format(self.label, key)) for key in self.choice_flags.keys()]
```

Comment:
I'm removing `e2e_experimental_settings.py` since its purpose was to automate different e2e QA experiments. The code, however, is outdated and should rather be replaced by a simple shell script.